CONTRIBUTING.md

```diff
@@ -95,6 +95,32 @@ test('should work', async () => {
 
 Some common test helpers, e.g. `testDir`, `isBuild` or `editFile` are available in `packages/playground/testUtils.ts`.
 
+Note: test build environment uses a different default set of Vite config to skip transpilation during tests to make it faster. But this makes the js files not process by `esbuild` compared to the production environment.
```

Comment:
vite use esbuild minify. And do not minify ES lib output since that would remove pure annotations and break tree-shaking.  https://github.com/vitejs/vite/blob/7ddfadab77737f386859d01c6a2e57a8c3dd9064/packages/vite/src/node/plugins/esbuild.ts#L240-L250

Reply:
Ah I see, if that's the case then it may be good to add a small note then if you like.